src/Middleware/StaticFiles/src/LoggerExtensions.cs

```diff
@@ -58,4 +58,8 @@ public static void FileServed(this ILogger logger, string virtualPath, string ph
 
     [LoggerMessage(14, LogLevel.Debug, "The file transmission was cancelled", EventName = "WriteCancelled")]
     public static partial void WriteCancelled(this ILogger logger, Exception ex);
+
+    [LoggerMessage(16, LogLevel.Warning,
+        "The WebRootPath was not found: {webRootPath}. Static files may be unavailable.", EventName = "WebRootPathNotFound")]
```

Comment:
```suggestion
        "The WebRootPath was not found: {WebRootPath}. Static files may be unavailable.", EventName = "WebRootPathNotFound")]
```
These names in logs are pascal case.

Reply:
Does that cause some different behavior? Where are these accessible from?